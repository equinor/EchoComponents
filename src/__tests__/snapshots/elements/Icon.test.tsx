import React from 'react';
import renderer from 'react-test-renderer';
import { Icon } from '../../..';

it('renders Icon wrapper correctly', () => {
    const tree = renderer.create(<Icon name={'arrow_up'} color={'#000'} title={'Up'} />).toJSON();
    expect(tree).toMatchSnapshot();
});
