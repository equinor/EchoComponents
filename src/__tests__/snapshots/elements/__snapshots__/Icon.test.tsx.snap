// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Icon wrapper correctly 1`] = `
<svg
  aria-labelledby="eds-arrow_up-1"
  className="sc-idOhPF bqIcIz"
  fill="#000"
  height="24px"
  name={null}
  role="img"
  title="Up"
  viewBox="0 0 24 24"
  width="24px"
  xmlns="http://www.w3.org/2000/svg"
>
  <title
    id="eds-arrow_up-1"
  >
    Up
  </title>
  <path
    className="sc-dIUggk"
    clipRule="evenodd"
    d="M4 12l1.41 1.41L11 7.83V20h2V7.83l5.58 5.59L20 12l-8-8-8 8z"
    data-testid="eds-icon-path"
    fillRule="evenodd"
    height="24"
    size={null}
    transform={null}
  />
</svg>
`;
